cypress/e2e/with_mock_data/catalogueCategories.cy.ts

```diff
@@ -29,7 +29,7 @@ describe('Catalogue Category', () => {
 
     cy.findByRole('link', { name: 'Motion' }).click();
     cy.findByRole('link', { name: 'Motion' }).should('not.exist');
-    cy.findByText('Actuators').should('not.exist');
+    cy.findByText('Actuators').should('exist');
```

Comment:
I don't think the assertion here makes sense anymore. It was checking Actuators is visible, then clicking a link and checking it doesn't exist anymore. Where as now its asserting it is there both before and after - so maybe it only needs the Motion one?

Reply:
there is no harm in including this test. checking for actuators to exist. tells us that the page has fully loaded.

Reply:
True but it doesn't actually do anything now considering it already has a check for Motion not existing as  a link but being visible elsewhere.